### PR TITLE
Added character, trump and player class

### PR DESCRIPTION
**Pull request by Kaspermn33**

### CODE REVIEW

File: src/worldofzuul/Character.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ * and open the template in the editor.
+ */
+package worldofzuul;
+
+/**
+ *
+ * @author Kasper
+ */
+public class Character {
+    private String name;
```
Comment (AndersBensen) [inline src/worldofzuul/Character.java]: add docblock for name attribute

File: src/worldofzuul/Trump.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ * and open the template in the editor.
+ */
+package worldofzuul;
+
+/**
+ *
+ * @author Kasper
+ */
+public class Trump extends Character {
+    String name;
```
Comment (AndersBensen) [inline src/worldofzuul/Trump.java]: Add docblock for attribute

Comment (quartercastle) [reply src/worldofzuul/Trump.java]: the attribute should be private.

File: src/worldofzuul/Character.java (hunk 1 above)
Comment (Kaspermn33) [reply src/worldofzuul/Character.java]: Fixed in eb0e8f2df038fa3aa4337affce65ba336a22da35

File: src/worldofzuul/Trump.java (hunk 2 above)
Comment (Kaspermn33) [reply src/worldofzuul/Trump.java]: Fixed both in 2c64b148ac86de64b48b13ee22f6b153531114b3

File: src/worldofzuul/Trump.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+ *
+ * @author Kasper
+ */
+public class Trump extends Character {
+    /**
+     * The name of the character
+     */
+    private String name;
+    
+    public Trump(){
```
Comment (quartercastle) [inline src/worldofzuul/Trump.java]: Should also have a docblock

File: src/worldofzuul/Character.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    
+    public Character(String name){
```
Comment (quartercastle) [inline src/worldofzuul/Character.java]: Should have a docblock

File: src/worldofzuul/Trump.java (hunk 3 above)
Comment (quartercastle) [reply src/worldofzuul/Trump.java]: `Trump.java` could be written as:
```java
public class Trump extends Character {
  private String name = "Donald Trump";
}
```

Comment (Kaspermn33) [reply src/worldofzuul/Trump.java]: Fixed in f6966ef8ef659f6045729267b7289b226522252e

**Review by quartercastle — APPROVED** [review]

**Pull request merged**